src/wallet/wallet.cpp

```diff
@@ -5232,7 +5232,7 @@ bool CWallet::AutoBackupWallet(const fs::path& wallet_path, bilingual_str& error
         }
         if(fs::exists(sourceFile)) {
             try {
-                fs::copy_file(sourceFile, backupFile);
+                fs::copy_file(sourceFile, backupFile, fs::copy_option::fail_if_exists);
```

Comment:
Because `fs::copy_file` is constrained [here](https://github.com/dashpay/dash/commit/834083fd9b64ace48057a977151fe2edee288ee5#diff-3492213c413b2d41f5e11a676db8f69ff54f71a2f5a14b2f61ee919d3a9f5122R95-R101), effectively excluding the variant that doesn't ask for `copy_options`. Without the change, there would be a compiler error.

```
wallet/wallet.cpp:5235:17: error: too few arguments to function call, expected 3, have 2; did you mean 'boost::filesystem::copy_file'?
                fs::copy_file(sourceFile, backupFile);
                ^~
                boost::filesystem::copy_file
/src/dash/depends/x86_64-pc-linux-gnu/include/boost/filesystem/operations.hpp:392:13: note: 'boost::filesystem::copy_file' declared here
inline bool copy_file(path const& from, path const& to)
            ^
1 error generated.
make[2]: *** [Makefile:12507: wallet/libbitcoin_wallet_a-wallet.o] Error 1
make[2]: Leaving directory '/src/dash/src'
make[1]: *** [Makefile:18776: all-recursive] Error 1
make[1]: Leaving directory '/src/dash/src'
make: *** [Makefile:804: all-recursive] Error 1
```